src/components/NewbieSelect/__tests__/NewbieSelect.test.tsx

```diff
@@ -8,11 +8,11 @@ import NewbieSelect from '../NewbieSelect';
 jest.mock('buddy-app-schema', () => {});
```

Comment:
It's gonna be unnecessary after the merge of the [PR-39](https://github.com/BuildIt-Poland/buddy-app/pull/39)